apps/condo/domains/common/utils/phone.spec.js

```diff
@@ -11,11 +11,19 @@ describe('normalizePhone()', () => {
 
     test('real phone +7', () => {
         expect(normalizePhone('+7 (906) 808 88 88')).toEqual('+79068088888')
+        expect(normalizePhone('+7 (906) 808 88 88', true)).toEqual('+79068088888')
         expect(normalizePhone('(906) 808 88 88')).toBeUndefined()
         expect(normalizePhone('8 (906) 808 88 88')).toBeUndefined()
         expect(normalizePhone('+7 (343) 808 88 88')).toBeUndefined()
     })
 
+    test('real phone +7 landline', () => {
```

Comment:
Could you please add a test that you can't register with `+7 (343) 808 88 88` that phone.

Reply:
done